---
title: Curriculum Vitae
url: cv.html
save_as: cv.html
slug: cv
lang: en
rank: 2
---

Nils Giordano, PhD
==================

---

> **Research Scientist** -- **Computational Systems Biologist**  
> *Data Analysis* -- *Mathematical Modeling* -- *Next-Generation Sequencing (NGS)* -- *Single-cell* -- *Omics*

---

Skills
----------------

Bioinformatics
:   Scripting (Python/Bash), Version control (Git), Pipeline design (Snakemake, contributor), Cluster usage (SGE/Slurm),
    Cloud computing (AWS), Metagenomics, scMultiomics (10x Genomics), Genome binning, Genome annotation (EggNOG/KO),
    Metabolic reconstruction (CarveMe), Metabolic modeling (Cobra)
&nbsp;

Biomathematics
:   Network analysis, Optimal control, Optimization, Metabolic Modeling,
    Linear/non-linear dynamical systems, 
    Time-series analysis (Kalman filtering/smoothing), 
    Image analysis (segmentation/tracking), Sensitivity analysis
&nbsp;

Microbiology
:   Cloning (molecular design, overlap PCR, Gibson assembly), 
    Chromosome editing (electroporation, lambda-red recombineering), 
    Microplate reader, Fluorescence microscope, Microfluidic
&nbsp;

Languages
:   French (native), English (fluent), Italian (basic)
&nbsp;

Research experience
-------------------

Since Jan. 2023
:   **Postdoctoral Researcher
    at INSERM (Nantes, France)**
    
    *Mentor: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Eric.Letouze@gmail.com)
    Dr. Éric Letouzé
    ([ICAGEN team](https://crci2na.univ-nantes.fr/en/research/team-8), CRCI<sup>2</sup>NA, INSERM U1307)*
    
    * Multiple Myeloma · Single Cell Multiomics · Next-Generation Sequencing (NGS)


Apr. 2020 - Jun. 2022
:   **Research Scientist
    at Integrative Phenomics (Paris, France)**
    
    * Developed pipelines for NGS analysis of gut metagenomic data and deployed them on the cloud (AWS)
    * Designed a metabolic modeling platform to predict how metabolites from different dietary options are processed by the microbiote
    * Analyzed blood and stool metabolomics data and compared them to model predictions
    * Produced scientific reports, co-wrote publications, and animated internal/external meetings
    * Participated in grant applications by designing and redacting work packages


Jan. 2018 - Dec. 2019
:   **Postdoctoral Researcher
    at Université de Nantes (Nantes, France)**
    
    *Mentor: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Samuel.Chaffron@ls2n.fr)
    Dr. Samuel Chaffron
    ([COMBI team](https://www.ls2n.fr/equipe/combi/), LS2N, CNRS)*
    
    * Constructed a global network of microbial interactions in the global ocean from Tara expeditions metagenomic and metatranscriptomic data
    * Identified communities of co-active cultivated and non-cultivated microbes (bacteria/archaea)
    * Uncovered functional traits linked to mutualism, including cross-feeding interactions derived from metabolic network reconstruction (publication in prep.)
    * Developed pipelines for NGS analysis of ocean metatranscriptomic data and deployed them on a cluster
    * Co-wrote and obtained a grant application for a 6-month extension of a Postdoc fellowship
    * Co-wrote publications and talked at several scientific conferences


Sep. 2012 - Mar. 2017
:   **Research and Teaching Assistant (PhD Student)
    at Université Grenoble Alpes -- Inria (Grenoble, France)**
    
    *Supervisors: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Hidde.de-Jong@inria.fr)
    Dr. Hidde de Jong
    ([Project-team Ibis](https://team.inria.fr/ibis/), Inria)
    & 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:hans.geiselmann@univ-grenoble-alpes.fr)
    Pr. Johannes Geiselmann
    ([team BIOP](http://www-liphy.ujf-grenoble.fr/Introduction-627), LIPhy)*
    
    * Constructed an abstract mathematical model of nutrient allocation in a microorganism
    * Applied Optimal Control to predict the optimal regulation of nutrient allocation during an environmental change
    * Showed that such a regulation is reminiscent of known regulatory processes in E. coli (published in Plos Comp. Biol.)
    * Engineered bacteria with fluorescent ribosomes and monitored them during an environmental change using a microfluidic device

### Internships

Feb.-Jun. 2012
:   **Research Assistant (Intern)
    at Inria (Grenoble, France)**

    *Supervisors: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Hidde.de-Jong@inria.fr)
    Dr. Hidde de Jong
    & 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Delphine.Ropers@inria.fr)
    Dr. Delphine Ropers
    ([Project-team Ibis](https://team.inria.fr/ibis/), Inria)*
    
    * Reviewed and implemented state-of-the-art methods of sensitivity analysis on a complex
    model of the gene expression machinery in bacterium *E. coli*

    * Developed a brand-new dynamical method of global sensitivity analysis

    * Helped to identify the key parameters driving the model dynamics and to reduce its complexity

Feb.-Jun. 2011
:   **Research Assistant (Intern)
    at University of Cambridge -- DAMTP (Cambridge, United Kingdom)**

    *Supervisor: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:R.E.Goldstein@damtp.cam.ac.uk)
    Pr. Raymond E. Goldstein
    ([Goldstein lab](http://www.damtp.cam.ac.uk/user/gold/), University of Cambridge)*

    * Led a theoretical study about the evolution towards multicellularity in microalgae (Volvocales)

    * Developed a general model of phosphate uptake and growth in microalgae

    * Explored the role of the extracellular matrix for phosphate storage, especially
    in changing environments

Jun.-Jul. 2010
:   **Research Assistant (Intern)
    at École Normale Supérieure (Paris, France)**

    *Supervisor:
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:demonte@biologie.ens.fr)
    Dr. Silvia de Monte
    ([Eco-evolutionary Mathematics](https://www.biologie.ens.fr/eceem/demonte.html), IBENS)*

    * Analyzed time-series fluorescence data of oscillating yeast cells suspensions
    subject to periodic forcing

    * Modified an existing mathematical model based on Hopf bifurcation to recreate the observed
    dynamics

    * Showed that even when an irrational forcing is applied,
    the biological system does not exhibit any chaotic behavior
    (published in Journal of Computational Interdisciplinary Sciences)
    
 Publications
------------

2024
:   **Single-cell data reveal heterogeneity of resource allocation across a bacterial population**  
    *Pavlou A, Cinquemani E, Pinel C, <u>Giordano N</u>, Van Melle Gateau M, Mihalcescu I, Geiselmann J, de Jong H*  
    *Submitted.*

2024
:   **Genome-scale community modelling reveals conserved metabolic cross-feedings in epipelagic bacterioplankton communities**  
    *<u>Giordano N,</u> Gaudin M, Trottier C, Delage E, Nef C, Bowler C, Chaffron S*  
    *In press. Nat. Commun. 2024*

2023
:   **Acquired resistance to a GPRC5D-directed T-cell engager in multiple myeloma is mediated by genetic or epigenetic target inactivation**  
    *Derrien J, Gastineau S, Frigout A, <u>Giordano N</u>, Cherkaoui M, Gaborit V, Boinon R, Douillard E, Devic M, Magrangeas F, Moreau P, Minvielle S, Touzeau C, Letouzé E*  
    Nat Cancer 2023. Aug 31;1-8.
    <a href="https://doi.org/10.1038/s43018-023-00625-9">
    <i class="ai ai-doi ai"></i></a>
    <a href="https://www.nature.com/articles/s43018-023-00625-9">
    <i class="fa fa-file-pdf-o fa"></i></a>

2022
:   **Characterization of the Gut Microbiota in Individuals with Overweight or Obesity during a Real-World Weight Loss Dietary Program: A Focus on the Bacteroides 2 Enterotype**  
    *Alili R, Belda E, Fabre O, Pelloux V, <u>Giordano N</u>, Legrand R, Bel Lassen P, Swartz TD, Zucker J-D, Clément K*  
    Biomedicines. 2022; 10(1):16.
    <a href="https://doi.org/10.3390/biomedicines10010016">
    <i class="ai ai-doi ai"></i></a>
    <a href="https://www.mdpi.com/2227-9059/10/1/16/pdf">
    <i class="fa fa-file-pdf-o fa"></i></a> 

2017
:   **Mathematical Modeling of Microbes:
    Metabolism, Gene Expression, and Growth**  
    *de Jong H, Casagranda S, <u>Giordano N</u>, Cinquemani E, Ropers D, Geiselmann J, Gouzé J-L*  
    Journal of The Royal Society Interface. 14.136 (2017): 20170502.
    <a href="http://dx.doi.org/10.1098/rsif.2017.0502">
    <i class="ai ai-doi ai"></i></a>
    <a href="http://rsif.royalsocietypublishing.org/content/14/136/20170502#aHR0cDovL3JzaWYucm95YWxzb2NpZXR5cHVibGlzaGluZy5vcmcvY29udGVudC9yb3lpbnRlcmZhY2UvMTQvMTM2LzIwMTcwNTAyLmZ1bGwucGRmQEBAMA==">
    <i class="fa fa-file-pdf-o fa"></i></a> 

2016
:   <i class="ai ai-open-access"></i>**Dynamical allocation of cellular resources as an optimal control problem:
    Novel insights into microbial growth strategies.**  
    *<u>Giordano N</u>, Mairet F, Gouzé J-L, Geiselmann J, de Jong H.*  
    PLoS Computational Biology. 2016;12(3):e1004802.
    <a href="http://dx.doi.org/10.1371/journal.pcbi.1004802">
    <i class="ai ai-doi ai"></i></a>
    <a href="http://journals.plos.org/ploscompbiol/article/asset?id=10.1371%2Fjournal.pcbi.1004802.PDF">
    <i class="fa fa-file-pdf-o fa"></i></a> 

2012
:   <i class="ai ai-open-access"></i>**Dynamical responses of oscillating yeast cells suspensions to periodic forcing.**  
    *<u>Giordano N</u>, D’Ovidio F, Danø S, Sørensen PG, De Monte S.*  
    Journal of Computational Interdisciplinary Sciences. 2012;3(2):77–86.
    <a href="http://dx.doi.org/10.6062/jcis.2012.03.02.0053">
    <i class="ai ai-doi ai"></i></a>
    <a href="http://epacis.net/jcis/PDF_JCIS/JCIS11-art.53.pdf">
    <i class="fa fa-file-pdf-o fa"></i></a> 

Other research activities
-------------------------

* Reviewer for European Control Conference 2019 ([ECC19](https://ecc19.eu/))


Teaching experience
-------------------

July 2018
:    * **Environmental Genomics**, Université d’Angers, 3 hours  
*Lecture and practical exercices for PhD students and young scientists at the Angers Bioinformatics Summer School,
in collaboration with Dr. Samuel Chaffron.*


Sept. 2013 to Aug. 2016
:   **Teaching assistant**; Université Grenoble Alpes (Grenoble, France), 229 hours

    * **Bioinformatics: from genome analysis to modeling (BSc)**  
*This introductory course was taught to 2nd-year students in collaboration with
Dr. Thierry Gautier.
I was in charge of the part about modeling, which means I was responsible for choosing 
the content, designing and teaching the lectures, tutorials, and practical exercices,
but also designing the final exam and grading.*

    * **Population genetics, conservation biology, biodiversity and evolution (BSc and MSc)**  
*These different courses were taught from 2nd-year to 4th-year students.
I mostly supervized tutorials and helped to design and grade the final exams.*

    * **Prokaryotic genetics and microbiology (BSc)**  
*This course was taught to 2nd-year students.
I supervized tutorials and practical exercices at the bench.*

    * **Modeling and simulation of genetic regulatory networks (MSc)**  
*This course was taught to 5th-year students.
It was organized for students of École Normale Supérieure, and taught in collaboration
with Dr. Hidde de Jong.
I helped to choose the content, teach the lectures and supervize the practical exercices.
I also helped to design and grad the final exam.*

Education
---------

2017
:   **PhD, Systems Biology** -- Université Grenoble Alpes (Grenoble, France)  

    * Title:  *Microbial growth control in changing environments:
    Theoretical and experimental study of resource allocation in Escherichia coli*  
    * Supervisors: *Dr. Hidde de Jong & Pr. Johannes Geiselmann*  
    * Laboratories: *[Project-team Ibis](https://team.inria.fr/ibis/) (Inria Grenoble -- Rhône-Alpes)
    and [team BIOP](http://www-liphy.ujf-grenoble.fr/Introduction-627)
    (Laboratoire Interdisciplinaire de Physique, Université Grenoble Alpes)*  

2012
:   **MSc, Cell Systems Biology** with honours -- École Normale Supérieure
    & Université Pierre et Marie Curie (Paris, France)  

    * 2nd year: Gene regulatory networks, cell ecosystems, cell machinery, microscopy (quantification,
    image processing), graph theory, comparative genomics, population genetics,
    structural bioinformatics, modeling of biopolymers
    
    * 1st year: *In silico* biology, genes and genomes, from gene to function, ecological systems biology,
    biophysics, biomathematics

2010
:   **BSc, Life sciences** with honours -- École Normale Supérieure
    & Université Pierre et Marie Curie (Paris, France)

    * Experimental practice, ecology, physiology, molecular and cellular biology,
    genetics and epigenetics, bioinformatics, statistics, biomathematics,
    statistical and optical physics, weak-bond chemistry, modeling in biology

2009-2013
:   **ENS Diploma** -- École Normale Supérieure (Paris, France)  

    * French leading research school, selected via a national competitive exam  

Misc -- Extracurricular activities
--------------------------

* Outdoor and indoor climbing and bouldering (6A/B outdoor, 6B/C indoor, french quotations)
* Advocate for the open-source and libre cultures
* Former contributor on [bioinfo-fr.net](https://bioinfo-fr.net),
    a communautary French blog for bioinformaticians. [List of articles](https://bioinfo-fr.net/author/lins) (French).
* Favorite keyboard layout: [BÉPO](https://en.wikipedia.org/wiki/B%C3%89PO)
* Favorite Linux distribution: [Debian Sid](https://wiki.debian.org/DebianUnstable)
* Favorite hardware brand: AMD (CPU and GPU)

----    
