---
Title: Curriculum Vitae
Url: cv.html
Save_as: cv.html
Slug: cv
Lang: en
Rank: 2
---

Nils Giordano, PhD
==================

---

> Computational Systems Biologist with interests in Evolution and Bioengineering   
> Fluent in Biology, Maths, and Computer Sciences  
> Love smart code, elegant equations, and Life's fundamental laws

---

Research experience
-------------------

Sep. 2012 - Mar. 2017
:   **Research and Teaching Assistant (PhD Student)
    at Université Grenoble Alpes -- Inria (Grenoble, France)**
    
    *Supervisors: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Hidde.de-Jong@inria.fr)
    Dr. Hidde de Jong
    ([Project-team Ibis](https://team.inria.fr/ibis/), Inria)
    & 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:hans.geiselmann@univ-grenoble-alpes.fr)
    Pr. Johannes Geiselmann
    ([team BIOP](http://www-liphy.ujf-grenoble.fr/Introduction-627), LIPhy)*
    
    * Led a theoretical and experimental study of resource allocations in bacterium *E. coli*

    * Used Optimal Control Theory principles to show that unexpected resource allocation behaviors can be
    predicted by assuming that bacteria optimize growth (published in Plos Comp. Biol.)

    * Engineered and monitored cells with fluorescent ribosomes using a microplate reader, microscope and
    microfluidic device

Feb.-Jun. 2012
:   **Research Assistant (Intern)
    at Inria (Grenoble, France)**

    *Supervisors: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Hidde.de-Jong@inria.fr)
    Dr. Hidde de Jong
    & 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:Delphine.Ropers@inria.fr)
    Dr. Delphine Ropers
    ([Project-team Ibis](https://team.inria.fr/ibis/), Inria)*
    
    * Reviewed and implemented state-of-the-art methods of sensitivity analysis on a complex
    model of the gene expression machinery in bacterium *E. coli*

    * Developed a brand-new dynamical method of global sensitivity analysis

    * Helped to identify the key parameters driving the model dynamics and to reduce its complexity

Feb.-Jun. 2011
:   **Research Assistant (Intern)
    at University of Cambridge -- DAMTP (Cambridge, United Kingdom)**

    *Supervisor: 
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:R.E.Goldstein@damtp.cam.ac.uk)
    Pr. Raymond E. Goldstein
    ([Goldstein lab](http://www.damtp.cam.ac.uk/user/gold/), University of Cambridge)*

    * Led a theoretical study about the evolution towards multicellularity in microalgae (Volvocales)

    * Developed a general model of phosphate uptake and growth in microalgae

    * Explored the role of the extracellular matrix for phosphate storage, especially
    in changing environments

Jun.-Jul. 2010
:   **Research Assistant (Intern)
    at École Normale Supérieure (Paris, France)**

    *Supervisor:
    [<i class="fa fa-envelope-o" aria-hidden="true"></i>](mailto:demonte@biologie.ens.fr)
    Dr. Silvia de Monte
    ([Eco-evolutionary Mathematics](https://www.biologie.ens.fr/eceem/demonte.html), IBENS)*

    * Analyzed time-series fluorescence data of oscillating yeast cells suspensions
    subject to periodic forcing

    * Modified an existing mathematical model based on Hopf bifurcation to recreate the observed
    dynamics

    * Showed that even when an irrational forcing is applied,
    the biological system does not exhibit any chaotic behavior
    (published in Journal of Computational Interdisciplinary Sciences)

Teaching experience
-------------------

Sept. 2013-Aug. 2016
:   **Teaching assistant**; Université Grenoble Alpes (Grenoble, France), 229 hours over 3 years

    * **Bioinformatics: from genome analysis to modeling (BSc)**  
*This introductory course was taught to 2nd-year students in collaboration with
Dr. Thierry Gautier.
I was in charge of the part about modeling, which means I was responsible for choosing 
the content, designing and teaching the lectures, tutorials, and practical exercices,
but also designing the final exam and grading.*

    * **Population genetics, conservation biology, biodiversity and evolution (BSc and MSc)**  
*These different courses were taught from 2nd-year to 4th-year students.
I mostly supervized tutorials and helped to design and grade the final exams.*

    * **Procaryotic genetics and microbiology (BSc)**  
*This course was taught to 2nd-year students.
I supervized tutorials and practical exercices at the bench.*

    * **Modeling and simulation of genetic regulatory networks (MSc)**  
*This course was taught to 5th-year students.
It was organized for students of École Normale Supérieure, and taught in collaboration
with Dr. Hidde de Jong.
I helped to choose the content, teach the lectures and supervize the practical exercices.
I also helped to design and grad the final exam.*

Education
---------

2017
:   **PhD, Cell Biology** -- Université Grenoble Alpes (Grenoble, France)  

    * Title:  *Microbial growth control in changing environments:
    Theoretical and experimental study of resource allocation in Escherichia coli*  
    * Supervisors: *Dr. Hidde de Jong & Pr. Johannes Geiselmann*  
    * Laboratories: *[Project-team Ibis](https://team.inria.fr/ibis/) (Inria Grenoble -- Rhône-Alpes)
    and [team BIOP](http://www-liphy.ujf-grenoble.fr/Introduction-627)
    (Laboratoire Interdisciplinaire de Physique, Université Grenoble Alpes)*  

2012
:   **MSc, Cell Systems Biology** with honours -- École Normale Supérieure
    & Université Pierre et Marie Curie (Paris, France)  

    * 2nd year: Gene regulatory networks, cell ecosystems, cell machinery, microscopy (quantification,
    image processing), graph theory, comparative genomics, population genetics,
    structural bioinformatics, modeling of biopolymers
    
    * 1st year: *In silico* biology, genes and genomes, from gene to function, ecological systems biology,
    biophysics, biomathematics

2010
:   **BSc, Life sciences** with honours -- École Normale Supérieure
    & Université Pierre et Marie Curie (Paris, France)

    * Experimental practice, ecology, physiology, molecular and cellular biology,
    genetics and epigenetics, bioinformatics, statistics, biomathematics,
    statistical and optical physics, weak-bond chemistry, modeling in biology

2009-2013
:   **ENS Diploma** -- École Normale Supérieure (Paris, France)  

    * French leading research school, selected via a national competitive exam  

Publications
------------

2016
:   <i class="ai ai-open-access"></i>**Dynamical allocation of cellular resources as an optimal control problem:
    Novel insights into microbial growth strategies.**  
    *Giordano N, Mairet F, Gouzé J-L, Geiselmann J, de Jong H.*  
    PLoS Computational Biology. 2016;12(3):e1004802.
    <a href="http://dx.doi.org/10.1371/journal.pcbi.1004802">
    <i class="ai ai-doi ai"></i></a>
    <a href="http://journals.plos.org/ploscompbiol/article/asset?id=10.1371%2Fjournal.pcbi.1004802.PDF">
    <i class="fa fa-file-pdf-o fa"></i></a> 

2012
:   <i class="ai ai-open-access"></i>**Dynamical responses of oscillating yeast cells suspensions to periodic forcing.**  
    *Giordano N, D’Ovidio F, Danø S, Sørensen PG, De Monte S.*  
    Journal of Computational Interdisciplinary Sciences. 2012;3(2):77–86.
    <a href="http://dx.doi.org/10.6062/jcis.2012.03.02.0053">
    <i class="ai ai-doi ai"></i></a>
    <a href="http://epacis.net/jcis/PDF_JCIS/JCIS11-art.53.pdf">
    <i class="fa fa-file-pdf-o fa"></i></a> 

Conferences & Workshops
-----------------------

2016
: Presentation in a workshop on Systemic and Symbolic Biology (BIOSS, Lyon, France)

2015
: Invited for a presentation in a seminar on the Modeling of Life (SeMoVi, Grenoble, France)

2015
: Selected for a presentation at the Advanced Lecture Course on Computational Systems Biology (CompSysBio, Aussois, France)

2015
: Participation in the workshop Integrative Cell Models at Lorentz Center (Leiden, The Netherlands)

2014
: Contribution in the 21st International Symposium on Mathematical Theory of Networks and Systems (MTNS 2014, Groningen, Netherlands)
    <a href="https://hal.inria.fr/hal-01079299/">
    <i class="fa fa-file-pdf-o fa"></i></a>

Technical skills
----------------

###Computing

* Daily user of Python, Git, and Unix/shell scripting.

* Experienced with Matlab/Octave, Scilab, R, PHP, SQL, HTML/CSS.

* Curious about Julia, Ruby, Rust

###Mathematics

* Proficient in data analysis and modeling, in particular linear and non-linear dynamical systems.

* Familiar with time-series analysis, image analysis, sensitivity analysis.
More recently Kalman filtering, calculus of variations and optimal control theory.

* Curious of statistics, graph theory, molecular modeling, machine learning (deep-learning).

###Laboratory skills

Molecular biology
:   Cloning (molecular design, overlap PCR, Gibson assembly),
    chromosomic modifications (electroporation, lambda-red recombineering)

Laboratory Robotics
:   Microplate reader (Perkin Elmer, Tecan), fluorescence microscope (Zeiss),
    microfluidic device (Mother machine)

Languages
---------

French
:   Native speaker

English
:   Fluent

Italian
:   Basic


Extracurricular activities
--------------------------

* Outdoor and indoor climbings (~ 6A/B outdoor, 6B/C indoor, french quotations)
* Regular author on [bioinfo-fr.net](https://bioinfo-fr.net),
    a communautary French blog intended for young bioinformaticians
* Member of [RSG France -- JeBIF](https://jebif.fr/fr/),
    the French society for young bioinformaticians
* Advocate for the open-source and libre cultures

----    

> <script type="text/javascript">
<!--
emm='org';
em='normalesup';
nn='ano';
n='giord';
p='nils';
document.write('<a class="email" href="mailto:'
    + p + '.' + n + nn + '+cv@' + em + '.' + emm +'">'+p+'.'+n+nn+'@'+em+'.'+emm+'</a>');
//-->
</script>
<noscript>[Please enable javascript to see my email address]</noscript>  
> Marseille (France)

----
